api/campaigns/index.js

```diff
@@ -89,6 +90,23 @@ router.get('/camp/:id', (req, res) => {
     });
 });
 
+router.get('/:id/submissions', async (req, res) => {
+  
+  const { id } = req.params;
+
+  await ApplicationSubmissions.findAllByCampaignId(id)
+    .then(applicationSubmissions => {
+      if(applicationSubmissions.length > 0) {
+        res.status(200).json({ applicationSubmissions, error: null });
+      } else {
+        res.status(400).json({ message: "Submissions not found in the database" });
+      }
```

Comment:
@dacruz21 what do you think about responding based on emptiness versus just returning a potentially empty array of submissions? If no one's applied, then the request still went through fine and we're not looking for a particular resource.

Reply:
Agreed, I see it's already fixed :+1: